src/rgw/rgw_lc.cc

```diff
@@ -841,6 +841,11 @@ int RGWLC::handle_multipart_expiration(rgw::sal::Bucket* target,
         if (perfcounter) {
           perfcounter->inc(l_rgw_lc_abort_mpu, 1);
         }
+        ldpp_dout(wk->get_lc(), 1) << "DELETED: abort_multipart_upload"
```

Comment:
this should not be at log level 1;  the correct level is probably 5

Reply:
please apply this to subsequent log messages, as well

Reply:
During the actual use of the cluster, in order to prevent the log file from being too large, debug_rgw is set to 1.If the log level is set to 5, this log message will not be recorded in the log file, and the log level about Deleted log messages in the lc context is 2.